### PR TITLE
Oriba/add item command

### PR DESCRIPTION
**Pull request by oribarak**
## Description

Add a command for monday.com extension to quickly adding an item to pre-defined board

## Screencast
image

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/monday/src/lib/api.tsx
Comment (freak4pc) [inline extensions/monday/src/lib/api.tsx:89]:
```suggestion
export async function getQuickAddBoardAndUser(): Promise<BoardsResponse> {
```

or, pass the id along to make it a tad more reusable:

```suggestion
export async function getBoardAndUser(boardId: number): Promise<BoardsResponse> {
```

File: extensions/monday/src/addItemToDefaultBoard.tsx
Comment (freak4pc) [inline extensions/monday/src/addItemToDefaultBoard.tsx:67]:
```suggestion
```

File: extensions/monday/src/addItemToDefaultBoard.tsx
Comment (freak4pc) [inline extensions/monday/src/addItemToDefaultBoard.tsx:70]:
```suggestion
```

File: extensions/monday/src/addItemToDefaultBoard.tsx
Comment (freak4pc) [inline extensions/monday/src/addItemToDefaultBoard.tsx:73]:
```suggestion
```

File: extensions/monday/src/addItemToDefaultBoard.tsx
Comment (freak4pc) [inline extensions/monday/src/addItemToDefaultBoard.tsx:27]:
```suggestion
        getCachedQuickAddBoard(),
        getCachedUser(),
```

File: extensions/monday/src/addItemToDefaultBoard.tsx
Comment (freak4pc) [inline extensions/monday/src/addItemToDefaultBoard.tsx:15]: I think calling the feature "Quick-add item" will be nicer and more obvious to people :) WDYT? 

```suggestion
export default function QuickAddItem() {
```

File: extensions/monday/package.json
Comment (freak4pc) [inline extensions/monday/package.json:38]:
```suggestion
      "name": "quickAddItem",
      "title": "Quick-add an Item",
```

File: extensions/monday/package.json
Comment (freak4pc) [inline extensions/monday/package.json:48]:
```suggestion
          "name": "quickAddBoardId",
```

File: extensions/monday/package.json
Comment (freak4pc) [inline extensions/monday/package.json:37]:
```suggestion
      "name": "quickAddItem",
```

File: extensions/monday/package.json
Comment (freak4pc) [inline extensions/monday/package.json:52]:
```suggestion
          "description": "ID of the board to quick-add items to"
```

**Review by freak4pc — CHANGES_REQUESTED** [review]
Sooo cool! 🤩 
Thanks for the great PR and feature, this is super useful. 
I left a couple of very minor thoughts, lmk what you think :)

File: extensions/monday/src/addItemToDefaultBoard.tsx
Comment (oribarak) [reply extensions/monday/src/addItemToDefaultBoard.tsx:15]: good idea

**Comment by freak4pc** [comment]
LGTM :) 
Waiting for someone from Raycast team to approve (maybe @pernielsentikaer if he'll have the time :) )

**Review by pernielsentikaer — APPROVED** [review]
Awesome @oribarak 🔥 Let's get it merged! If you want to be added as a contributor, send me your raycast developer username 😄

**Comment by oribarak** [comment]
Thanks for the super quick review and merge :)

My developer username is oribarak, would love to join as a contributor

**Pull request merged**
